tests/unit_tests/ensemble_evaluator/test_evaluator_tracker.py

```diff
@@ -330,7 +330,7 @@ def test_tracking_progress(
 
     The final update event and end event is also tested."""
 
-    brm = run_model(None, None, None, None, None)
+    brm = run_model(None, None, None, None, None, None)
```

Comment:
Is this the reason for:
```python
if ert is not None:
```
that was present in the run models do you think? If so we can mock our way instead.